pgmpy/metrics/metrics.py

```diff
@@ -394,3 +395,44 @@ def fisher_c(model, data, ci_test, show_progress=True):
     C = -2 * np.log(cis.loc[:, "p_value"]).sum()
     p_value = 1 - stats.chi2.cdf(C, df=2 * cis.shape[0])
     return p_value
+
+
+def SHD(true_model, est_model):
+    """
+    Computes the Structural Hamming Distance between `true_model` and `est_model`.
```

Comment:
Ideally, I would structure the method explanation as: A single line about what the function does. Leave an empty line after and the give the full description. For example, I would change you current explanation as follows:
```
Computes the Structural Hamming Distance between `true_model` and `est_model`.

SHD is defined as total number of basic operations: adding edges, removing
edges, and reversing edges required to transform one graph to the other. It
is a symmetrical measure.
```